app/api-client.js

```diff
@@ -103,6 +104,7 @@ module.exports = (function () {
     return api_request({method: 'POST', endpoint: 'apps/', resource: app});
   };
 
+  // TODO: Rename `connect_bucket_to_app` => `grant_app_access_to_bucket`
```

Comment:
Don't leave TODOs in the code. Either do them, or add a ticket to Trello.

Reply:
😭

Reply:
Copy that: https://trello.com/c/nvOCtrlY/472-1-rename-appsconnectbucket-appsgrantbucketaccess 

FYI: @axemonkey I added that ticket, I could rename it myself later or you could do it if you feel like it as part of your current set of changes.